docs/tutorials/prerequisites/install-graphviz.md

```diff
@@ -0,0 +1,9 @@
+## Install Graphviz
+#### On Ubuntu
+```
+sudo apt-get install graphviz
```

Comment:
We are moving Graphviz installation instruction to the MXNet installation page. Once that is done, we can remove graphviz installation instructions from the tutorials.